pepys_import/import.py

```diff
@@ -25,9 +25,17 @@ def main(path=DIRECTORY_PATH, db=DEFAULT_DATABASE):
     parser = argparse.ArgumentParser()
     path_help = "The path to import data from (The default value is the directory of the script)"
     db_help = "The database to connect to: (The is to use an in-memory database)"
+    archive_help = "The flag to move files after the import (The default value is True)"
```

Comment:
this guidance doesn't quite match the implementation.  Suggest:
`Instruction to archive (move) imported files to designated archive folder`

Reply:
Yes, you're right. I wrote it for `--archive True/False`. I'm changing it.

Reply:
Help text changed:
https://github.com/debrief/pepys-import/pull/216/commits/a89b6de6c3fc129078b8acbcd2610cf87becdda8